CONTRIBUTING.md

```diff
@@ -10,15 +10,25 @@ Welcome! We are very happy to accept community contributions to Ratify, whether
 * Checkout the repo locally with `git clone git@github.com:{your_username}/ratify.git`.
 * Build the Ratify CLI with `go build -o ./bin/ratify ./cmd/ratify` or if on Mac/Linux/WSL `make build-cli`.
 
+
+## Feature Enhancements
+For non-trivial enhancements or bug fixes, please start by raising a design PR. You can refer to the example [here](https://github.com/ratify-project/ratify/blame/dev/docs/proposals/Release-Supply-Chain-Metadata.md).
+The design document should include:
+- The proposed changes
+- The impacted code paths
```

Comment:
Should we also add required sections such as: backwards compatibility, testing changes required, chart changes required, open question.

Reply:
added. thanks!